.github/workflows/test.yml

```diff
@@ -106,11 +106,14 @@ jobs:
           path: packages/validator/spec-tests
           key: spec-test-data-${{ hashFiles('packages/validator/test/spec/params.ts') }}
 
+      - name: Constants tests
+        run: yarn test:constants
```

Comment:
since we replaced `lerna run` with vitest run commands in https://github.com/ChainSafe/lodestar/pull/7610 we are no longer running those tests in the CI, seemed easiest to add them as a separate step, although run them as par of unit test command below